docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -143,6 +143,7 @@ To query sound device information:
       - `sound_manager_get_device_io_direction()`: To get the device IO direction.
       - `sound_manager_get_device_id()`: To get the device ID.
       - `sound_manager_get_device_name()`: To get the device name.
+      - `sound_manager_is_device_running()`: To check if the device is running which means any stream is running on the device.
```

Comment:
Suggestion:
Can we rephrase to:
To verify whether the device is running. This means that a stream is running on the device.

Please verify whether the technical meaning is altered..

Reply:
It looks fine, will be updated soon.